include/exadg/solvers_and_preconditioners/multigrid/transfers/mg_transfer_global_coarsening.cpp

```diff
@@ -111,7 +111,8 @@ MGTransferGlobalCoarsening<dim, Number, VectorType>::reinit(
   }
 
   mg_transfer_global_coarsening =
-    std::make_unique<dealii::MGTransferGlobalCoarsening<dim, VectorType>>(transfers);
+    std::make_unique<dealii::MGTransferGlobalCoarsening<dim, VectorType>>(
+      transfers, [&](const auto l, auto & vec) { mg_matrixfree[l]->initialize_dof_vector(vec); });
```

Comment:
@kronbichler This is all what is needed for the partitioners. Could you check with your benchmark that the vectors-copy issue is fixed now?

Reply:
Yes, I can confirm that we do not enter the slow path in `restrict_and_add` and `prolongate_and_add` with this change, while we did before.